src/V2/Records/ListRelatedQuery.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace Zoho\Crm\V2\Records;
+
+use Zoho\Crm\Contracts\PaginatedQueryInterface;
+use Zoho\Crm\Contracts\ResponseTransformerInterface;
+use Zoho\Crm\Support\Helper;
+use Zoho\Crm\V2\Traits\HasPagination;
+
+class ListRelatedQuery extends AbstractQuery implements PaginatedQueryInterface
+{
+    use HasPagination;
+
+    /**
+     * @var string The record id
+     */
+    protected $recordId;
+
+    /**
+     * @var string The related list API name
+     */
+    protected $relatedList;
+
+    /**
+     * @inheritdoc
+     */
+    public function getUrl(): string
+    {
+        return "$this->module/$this->recordId/$this->relatedList";
+    }
+
+    /**
+     * Get record Id.
+     */
+    public function getRecordId(): string
+    {
+        return $this->recordId;
+    }
+
+    /**
+     * Set record Id.
+     * @param string $id
+     */
+    public function setRecordId(string $id): void
+    {
+        $this->recordId = $id;
+    }
+
+    /**
+     * Get related list API name.
+     *
+     * @return string
+     */
+    public function getRelatedList(): string
+    {
+        return $this->relatedList;
+    }
+
+    /**
+     * Set related list API Name.
+     *
+     * @param string $relatedListName
+     */
+    public function setRelatedList(string $relatedListName): void
+    {
+        $this->relatedList = $relatedListName;
+    }
+
+    /**
+     * Set related list API name and record Id.
```

Comment:
"record **ID**"

Reply:
function removed and php block removed :+1: